ci/env.py

```diff
@@ -3,7 +3,7 @@
 import os
 
 LIBTELIO_ENV_MOOSE_RELEASE_TAG = "v0.10.1-meshnetApp"
-LIBTELIO_ENV_NAT_LAB_DEPS_TAG = "v0.0.21"
+LIBTELIO_ENV_NAT_LAB_DEPS_TAG = "llt-4310-reapply-ipv6-stun"
```

Comment:
Adding message to remind you to change this to real version tag :)

Reply:
still relevant :)

Reply:
I know, when I'll receive +2 on PRs, I'll merge those and update this :)

Reply:
done